docs/en/writing-running-appium/ios/ios-xctest-file-movement.md

```diff
@@ -25,12 +25,12 @@ $ brew install ifuse --HEAD # newer iOS versions need the latest codebase
 
 The format of method argument should be the following:
 
-- `@<app_bundle_id>` is a bundle id.
+- `@<app_bundle_id>` is a bundle id
 - `optional_container_type` is container type
-    - `documents` is only available. Others work as _format 2_
+    - `documents` is only available
         - BundleIds which can get by `ifuse -u <udid> --list-apps` can specify `documents` type
```

Comment:
you may specify `documents` container type only for bundle ids returned by  ... command